yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -141,15 +142,37 @@ func packAndSendAssessMigrationPayload(status string, errMsg error) {
 	if assessmentReport.TableIndexStats != nil {
 		for _, stat := range *assessmentReport.TableIndexStats {
 			newStat := callhome.ObjectSizingStats{
-				//redacting schema and object name
-				ObjectName:      constants.OBFUSCATE_STRING,
 				ReadsPerSecond:  utils.SafeDereferenceInt64(stat.ReadsPerSecond),
 				WritesPerSecond: utils.SafeDereferenceInt64(stat.WritesPerSecond),
 				SizeInBytes:     utils.SafeDereferenceInt64(stat.SizeInBytes),
 			}
+
+			// Anonymizing schema and object names
+			sname, err := anonymizer.AnonymizeSchemaName(stat.SchemaName)
```

Comment:
nit: we could have helper methods in cmd pkg that do two things: try to anonymize, if not, return the obfuscate_string.

Reply:
The concern is not about whether it's empty or not. 
Just a helper method so that we have less if-else code here repeated.

Reply:
hmm would have to write helpers for each case - sql, table, column etc..

Also would like to stick to this, since if-else here is due to error message handling only which is fine to be  explicit.